doc/documentation_data_format.rst

```diff
@@ -741,3 +741,118 @@ allows to specify multiple SBML models with corresponding condition and
 measurement tables, and one joint parameter table. This means that the parameter
 namespace is global. Therefore, parameters with the same ID in different models
 will be considered identical.
+
+
+Math expressions syntax
+-----------------------
+
+This section describes the syntax of math expressions used in PEtab files, such
+as, e.g., the observable formulas.
+
+General
+~~~~~~~
+
+* Whitespace is ignored in math expressions.
+* All parameter IDs from the parameter table, as well as all IDs of model
+  entities that are globally unique and match the PEtab identifier format
+  may be used in math expressions unless
+  stated otherwise.
+* Operators have to be specified. Implicit multiplication is not supported,
+  i.e., ``a b`` instead of ``a * b`` is not allowed.
+* It is not allowed to refer to any functions defined in the model.
+* Special symbols (such as :math:`e`, :math:`\pi`) are not supported, neither is NaN
+  (not-a-number)
+
+Operators
+~~~~~~~~~
+
+The supported operators are:
+
+.. csv-table:: Supported operators
+   :header: "Operator", "Precedence", "Interpretation", "Associativity", "Arguments", "Evaluates to"
+   :widths: 10, 10, 50, 15, 20, 15
+
+   "``f()``", "1", "function arguments", "left-to-right", "any", "input-dependent"
+   "``()``", "1", "parentheses for grouping, acts like identity", "", "any single expression", "argument"
+   "``^``", "2", "exponentiation (shorthand for pow)", "right-to-left", "float, float", "float"
+   "``+`` ``-``", "3", "unary plus/minus", "right-to-left", "float", "float"
+   "``!``", "3", "not", "", "bool", "bool"
+   "``*`` ``/``", "4", "multiplication division", "left-to-right", "float, float", "float"
+   "``+`` ``-``", "5", "binary plus/minus, addition/subtraction", "left-to-right", "float, float", "float"
+   "``<`` ``<=`` ``>`` ``>=``", "6", "less than, less than or equal to, greater than, greater than or equal to", "left-to-right", "float, float", "bool"
+   "``==`` ``!=``", "6", "is equal to, is not equal to", "left-to-right", "float, float or bool, bool", "bool"
+   "``&&``, ``||``", "7", "logical and, logical or", "left-to-right", "bool, bool", "bool"
+   "``,``", "8", "comma  as function argument separator", "left-to-right", "any", ""
```

Comment:
Not sure if this will work, but there are RST tables that support newlines, and they might improve the look of these tables, e.g. less horizontal scrolling. If it works, I can do the same for the functions table.

Reply:
Also in favor of line breaks, but I thought I tried and failed with the csv tables. I'll test it again.

Reply:
Such a pain... but done.